src/components/LooMap/useLocateMapControl.ts

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import L, { LatLngExpression, Map } from 'leaflet';
```

Comment:
```suggestion
import L, { LatLngLiteral, Map } from 'leaflet';
```

I don't think we were using any of the other types provided in the `LatLngExpression`, so i'm putting this (there are two successive changes that i can't put as gh code suggestions) in to narrow it down to just the `LatLngLiteral` type that you used in the `fitMapBoundsToUserLocationNeighbours` method signature so we can satisfy the type checker.

Reply:
worth mentioning that the Typescript migration is still a WIP, but we're trying to keep any new code typesafe so we reduce tech debt as we slowly squash all our type errors